packages/graph-retriever/src/graph_retriever/testing/adapter_tests.py

```diff
@@ -273,6 +314,10 @@ class AdapterComplianceSuite(abc.ABC):
     loaded.
     """
 
+    def supports_nested_metadata(self) -> bool:
+        """Return whether nested metadata is expected to work."""
```

Comment:
Add some ability to create "features" that disable respective tests. Can refine later.